contents/handbook/people/compensation.mdx

```diff
@@ -69,11 +72,23 @@ As we do these much more frequently than regular companies, team members should
 
 ### Pay review process
 
+Pay at PostHog is reviewed much more often at most other companies - at least 2 to 3 times a year.
+
+To make sure everyone has an equal chance at getting a payrise we do not factor in how much someone complains about their pay. When increasing pay we only look at our calculator and performance.
```

Comment:
Maybe something a bit softer here, such as "we do not factor in how often someone requests a pay adjustment"

Reply:
Agreed, updated